src/open-data/Services.csv

```diff
@@ -1,314 +1,314 @@
+Microsoft.Cdn;microsoft.cdn/profiles;Content Delivery Network;Networking;Microsoft;Cloud Provider;Cloud;SaaS
+microsoft.certificateregistration;microsoft.certificateregistration/certificateorders;Azure App Service;Web;Microsoft;Cloud Provider;Both;PaaS
+microsoft.classiccompute;microsoft.classiccompute/domainnames;Virtual Machines;Compute;Microsoft;Cloud Provider;Both;IaaS
+microsoft.classiccompute;microsoft.classiccompute/virtualmachines;Virtual Machines;Compute;Microsoft;Cloud Provider;Both;IaaS
+Microsoft.ClassicNetwork;microsoft.classicnetwork/ipv4vip;Virtual Network;Networking;Microsoft;Cloud Provider;Both;IaaS
+Microsoft.ClassicNetwork;microsoft.classicnetwork/publicip;Virtual Network;Networking;Microsoft;Cloud Provider;Both;IaaS
+Microsoft.ClassicNetwork;microsoft.classicnetwork/publicipaddresses;Virtual Network;Networking;Microsoft;Cloud Provider;Cloud;IaaS
+Microsoft.ClassicStorage;microsoft.classicstorage/storageaccounts;Storage Accounts;Storage;Microsoft;Cloud Provider;Cloud;IaaS
+Microsoft.CognitiveServices;microsoft.cognitiveservices/accounts;Azure AI Services;AI and Machine Learning;Microsoft;Cloud Provider;Cloud;SaaS
+microsoft.communication;microsoft.communication/communicationservices;Azure Communication Services;Web;Microsoft;Cloud Provider;Cloud;PaaS
+Microsoft.EventGrid;microsoft.communication/communicationservices;Azure Communication Services;Web;Microsoft;Cloud Provider;Cloud;PaaS
+Microsoft.Compute;microsoft.compute/capacityreservationgroups;Virtual Machines;Compute;Microsoft;Cloud Provider;Cloud;PaaS
+Microsoft.Compute;microsoft.compute/cloudservices;Cloud Services;Compute;Microsoft;Cloud Provider;Cloud;SaaS
+Microsoft.Compute;microsoft.compute/disks;Virtual Machines;Compute;Microsoft;Cloud Provider;Cloud;IaaS
+Microsoft.Compute;microsoft.compute/galleries;Virtual Machines;Compute;Microsoft;Cloud Provider;Cloud;IaaS
+Microsoft.Compute;microsoft.compute/hostgroups;Azure Dedicated Host;Compute;Microsoft;Cloud Provider;Cloud;IaaS
+Microsoft.Compute;microsoft.compute/images;Virtual Machines;Compute;Microsoft;Cloud Provider;Cloud;IaaS
+Microsoft.Compute;microsoft.compute/restorepointcollections;Virtual Machines;Compute;Microsoft;Cloud Provider;Cloud;IaaS
+Microsoft.Compute;microsoft.compute/sharedvmimages;Virtual Machines;Compute;Microsoft;Cloud Provider;Cloud;IaaS
+Microsoft.Compute;microsoft.compute/snapshots;Virtual Machines;Compute;Microsoft;Cloud Provider;Cloud;IaaS
+microsoft.compute;microsoft.compute/virtualmachines;Virtual Machines;Compute;Microsoft;Cloud Provider;Cloud;IaaS
+microsoft.compute;microsoft.compute/virtualmachinescalesets;Virtual Machine Scale Sets;Compute;Microsoft;Cloud Provider;Cloud;IaaS
+Microsoft.ConfidentialLedger;microsoft.confidentialledger/ledgers;Azure Confidential Ledger;Databases;Microsoft;Cloud Provider;Cloud;PaaS
+Microsoft.ContainerInstance;microsoft.containerinstance/containergroups;Azure Container Instances;Compute;Microsoft;Cloud Provider;Cloud;PaaS

```

Comment:
IaaS? 🤔 In this capacity, it's ultimately a VM, isn't it?

Reply:
Deepest level is IaaS, corrected